tests/test_pipeline_mixin.py

```diff
@@ -214,8 +214,6 @@ def run_task_tests(self, task, torch_dtype="float32"):
                     image_processor_names.append(cls_name)
                 elif "FeatureExtractor" in cls_name:
                     feature_extractor_names.append(cls_name)
-                else:
-                    raise ValueError(f"Unknown processor class: {cls_name}")
```

Comment:
`Pix2Struct` model has `Processor` object in tiny model json, and that led to an error raised for `Pix2Struct` and filed test. Error raise is not required here because we take the processor from `PROCESSOR_MAPPING_NAMES`, we can just skip this Processor